tests/languages/sbt_test.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from itertools import product
+from pathlib import Path
+from typing import Any
+
+import pytest
+
+from pre_commit.hook import Hook
+from pre_commit.languages import sbt
+from testing.util import cwd
+from testing.util import skipif_cant_run_sbt
+
+
+@skipif_cant_run_sbt
+@pytest.mark.parametrize(
+    ['args', 'files'],
+    product(
+        [
+            [], ['argfile1.txt'], ['argfile1.txt', 'argfile2.txt'],
+            ['\"arg file1.txt\"'], ['\"arg file1.txt\"', '\"arg file2.txt\"'],
+        ],
+        [
+            [], ['filesfile1.txt'], ['filesfile1.txt', 'filesfile2.txt'],
+            ['files file1.txt'], ['files file1.txt', 'files file2.txt'],
+        ],
+    ),
+)
+def test_sbt_hook(
+        sbt_project_with_touch_command: Path,
+        args: list[str],
+        files: list[str],
```

Comment:
[typing tests is a waste of time](https://youtu.be/lGCU0j-otls)

Reply:
I personally, find them useful (even for tests), but I do understand your perspective in the video.

In this comment are you asking me to remove the type annotations from test code, or just suggesting that it is not a requirement to use type annotations in test code? (I don't mind either way)

Reply:
yes please remove them unless they are necessary